### ISSUE TITLE
Cannot reproduce results

### ISSUE THREAD

**Issue by yzleroy** [zzh-tech/ESTRNN#16]
Hi, dear authors, I've recently reproduced your work, the command is as following:

       python main.py --data_root /home/data/ --dataset gopro_ds_lmdb --ds_config 2ms16ms --batch_size 32 --data_format RGB

I use 4 x p40 devices, but the output is:

2021/06/09, 02:28:12 - gopro_ds_lmdb results generating ...
2021/06/09, 02:28:12 - seq 000 image results generating ...
2021/06/09, 02:28:54 - seq 001 image results generating ...
2021/06/09, 02:29:33 - seq 002 image results generating ...
2021/06/09, 02:30:11 - seq 003 image results generating ...
2021/06/09, 02:30:49 - seq 004 image results generating ...
2021/06/09, 02:31:28 - seq 005 image results generating ...
2021/06/09, 02:32:06 - seq 006 image results generating ...
2021/06/09, 02:32:59 - seq 007 image results generating ...
2021/06/09, 02:33:37 - seq 008 image results generating ...
2021/06/09, 02:34:15 - seq 009 image results generating ...
2021/06/09, 02:34:53 - seq 010 image results generating ...

2021/06/09, 02:35:24 - Test images : 1067
2021/06/09, 02:35:24 - Test PSNR : 29.877435602139418
2021/06/09, 02:35:24 - Test SSIM : 0.8923279783187907
2021/06/09, 02:35:24 - Average time per image: 0.07301736338173642

Final PSNR and SSIM are much lower than paper's results.

**Comment by yzleroy** [comment]
Furthermore, can you provide pretrained weights on GoPro dataset?

**Comment by zzh-tech** [comment]
Hi, the batch size will affect the performance, if you keep the other hyper-parameters unchanged. (same learning rate, better performance with smaller batch size)

> Furthermore, can you provide pretrained weights on GoPro dataset?

Sorry, I can only provide the pretrained weights for our BSD dataset.
The pre-training weights for the GoPro dataset are in Huawei's workstation, and I don't have access to it.

**Comment by yzleroy** [comment]
> Hi, the batch size will affect the performance, if you keep the other hyper-parameters unchanged. (same learning rate, better performance with smaller batch size)
> 
> > Furthermore, can you provide pretrained weights on GoPro dataset?
> 
> Sorry, I can only provide the pretrained weights for our BSD dataset.
> The pre-training weights for the GoPro dataset are in Huawei's workstation, and I don't have access to it.

Dear author, thanks for your quick reply, and I want to know how to reproduce the results of GoPro dataset.

**Comment by zzh-tech** [comment]
Please keep the batch size as default value, i.e. 8.

**Comment by yzleroy** [comment]
> Please keep the batch size as default value, i.e. 8.

OK, I'll try your suggestion
